x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ClientHelper.java

```diff
@@ -196,6 +196,7 @@ private static String maybeRewriteSingleAuthenticationHeaderForVersion(
     public static final String APM_ORIGIN = "apm";
     public static final String OTEL_ORIGIN = "otel";
     public static final String REINDEX_DATA_STREAM_ORIGIN = "reindex_data_stream";
+    public static final String MIGRATE_ORIGIN = "migrate";
```

Comment:
Let's use the existing `STACK_ORIGIN` origin, I think there's an aversion to adding too many of these if I remember correctly.